src/components/display/fewlines/AwaitingValidationIcon/AwaitingValidationIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+export const AwaitingValidationIcon: React.FC = () => {
+  return (
+    <svg
+      width="14"
+      height="14"
+      viewBox="0 0 14 14"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M7.00004 0.333252C5.6815 0.333252 4.39257 0.724245 3.29624 1.45679C2.19991 2.18933 1.34543 3.23052 0.840847 4.4487C0.336263 5.66687 0.20424 7.00731 0.461475 8.30052C0.71871 9.59373 1.35365 10.7816 2.286 11.714C3.21835 12.6463 4.40623 13.2813 5.69944 13.5385C6.99265 13.7957 8.33309 13.6637 9.55127 13.1591C10.7694 12.6545 11.8106 11.8 12.5432 10.7037C13.2757 9.60739 13.6667 8.31846 13.6667 6.99992C13.6667 6.12444 13.4943 5.25753 13.1592 4.4487C12.8242 3.63986 12.3331 2.90493 11.7141 2.28587C11.095 1.66682 10.3601 1.17575 9.55127 0.840722C8.74243 0.50569 7.87552 0.333252 7.00004 0.333252ZM7.00004 12.3333C5.94521 12.3333 4.91406 12.0205 4.037 11.4344C3.15994 10.8484 2.47635 10.0154 2.07269 9.0409C1.66902 8.06636 1.5634 6.994 1.76919 5.95944C1.97498 4.92487 2.48293 3.97456 3.22881 3.22868C3.97469 2.4828 4.925 1.97485 5.95956 1.76906C6.99413 1.56328 8.06648 1.66889 9.04102 2.07256C10.0156 2.47623 10.8485 3.15982 11.4345 4.03688C12.0206 4.91394 12.3334 5.94509 12.3334 6.99992C12.3334 8.41441 11.7715 9.77096 10.7713 10.7712C9.77109 11.7713 8.41453 12.3333 7.00004 12.3333Z"
+        fill="white"
+      />
+      <path
+        d="M7.00004 10.3333C7.36823 10.3333 7.66671 10.0349 7.66671 9.66667C7.66671 9.29848 7.36823 9 7.00004 9C6.63185 9 6.33337 9.29848 6.33337 9.66667C6.33337 10.0349 6.63185 10.3333 7.00004 10.3333Z"
+        fill="white"
+      />
+      <path
+        d="M7.00004 3.6665C6.82323 3.6665 6.65366 3.73674 6.52864 3.86177C6.40361 3.98679 6.33337 4.15636 6.33337 4.33317V7.6665C6.33337 7.84331 6.40361 8.01288 6.52864 8.13791C6.65366 8.26293 6.82323 8.33317 7.00004 8.33317C7.17685 8.33317 7.34642 8.26293 7.47144 8.13791C7.59647 8.01288 7.66671 7.84331 7.66671 7.6665V4.33317C7.66671 4.15636 7.59647 3.98679 7.47144 3.86177C7.34642 3.73674 7.17685 3.6665 7.00004 3.6665Z"
+        fill="white"
+      />
+    </svg>
```

Comment:
I have you used `npx svgo` on the 2 svgs?

Reply:
No I didn't, now I did 11b5261

Reply:
Haha, see how effective it is? And how svgs can be bloated haha.